25.11.2022 17:37:06: Client Created successfully 
25.11.2022 17:37:35: Client Created successfully 
25.11.2022 17:37:40: Message: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.

StackTrace: 

Server stack trace: 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.CreateConnection () [0x00036] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.GetConnection () [0x000bb] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpConnectionPool.GetConnection (System.String host, System.Int32 port) [0x0005c] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpClientTransportSink.ProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg, System.Runtime.Remoting.Channels.ITransportHeaders requestHeaders, System.IO.Stream requestStream, System.Runtime.Remoting.Channels.ITransportHeaders& responseHeaders, System.IO.Stream& responseStream) [0x00025] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.BinaryClientFormatterSink.SyncProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg) [0x0006c] in <f4b659e5df5b40fda5b047a02f25c201>:0 

Exception rethrown at [0]: 
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_remoting_wrapper(intptr,intptr)
  at (wrapper remoting-invoke) IMarketingGameResult.IMarketingGameResult.Log(string)
  at Client.SendToServer (System.String[] data) [0x00017] in C:\Games\Rauf Tur\Assets\Scripts\Classes\Client.cs:16 
  at CommandLineArgsHandler.Start () [0x000c9] in C:\Games\Rauf Tur\Assets\CommandLineArgsHandler.cs:103 
Source: 
Date: System.Collections.ListDictionaryInternal
InnerException: 

25.11.2022 17:37:59: session closed successfully with response:StatusCode: 202, ReasonPhrase: 'Accepted', Version: 1.1, Content: System.Net.Http.StreamContent, Headers:
{
Server: nginx/1.23.1
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/7.4.25
Cache-Control: no-cache, private
Date: Fri, 25 Nov 2022 15:37:59 GMT
X-Debug-Token: 96f019
X-Debug-Token-Link: http://212.56.202.77/_profiler/96f019
X-Robots-Tag: noindex
Content-Type: text/html; charset=UTF-8
}
25.11.2022 17:38:03: Message: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.

StackTrace: 

Server stack trace: 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.CreateConnection () [0x00036] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.GetConnection () [0x000bb] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpConnectionPool.GetConnection (System.String host, System.Int32 port) [0x0005c] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpClientTransportSink.ProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg, System.Runtime.Remoting.Channels.ITransportHeaders requestHeaders, System.IO.Stream requestStream, System.Runtime.Remoting.Channels.ITransportHeaders& responseHeaders, System.IO.Stream& responseStream) [0x00025] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.BinaryClientFormatterSink.SyncProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg) [0x0006c] in <f4b659e5df5b40fda5b047a02f25c201>:0 

Exception rethrown at [0]: 
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_remoting_wrapper(intptr,intptr)
  at (wrapper remoting-invoke) IMarketingGameResult.IMarketingGameResult.Prize(string)
  at Client.SendToServer (System.String[] data) [0x0003c] in C:\Games\Rauf Tur\Assets\Scripts\Classes\Client.cs:17 
  at QuitGame.Update () [0x000b5] in C:\Games\Rauf Tur\Assets\Scripts\QuitGame.cs:48 
Source: 
Date: System.Collections.ListDictionaryInternal
InnerException: 

25.11.2022 17:58:38: Client Created successfully 
25.11.2022 17:58:43: Message: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.

StackTrace: 

Server stack trace: 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.CreateConnection () [0x00036] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.GetConnection () [0x000bb] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpConnectionPool.GetConnection (System.String host, System.Int32 port) [0x0005c] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpClientTransportSink.ProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg, System.Runtime.Remoting.Channels.ITransportHeaders requestHeaders, System.IO.Stream requestStream, System.Runtime.Remoting.Channels.ITransportHeaders& responseHeaders, System.IO.Stream& responseStream) [0x00025] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.BinaryClientFormatterSink.SyncProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg) [0x0006c] in <f4b659e5df5b40fda5b047a02f25c201>:0 

Exception rethrown at [0]: 
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_remoting_wrapper(intptr,intptr)
  at (wrapper remoting-invoke) IMarketingGameResult.IMarketingGameResult.Log(string)
  at Client.SendToServer (System.String[] data) [0x00017] in C:\Games\Rauf Tur\Assets\Scripts\Classes\Client.cs:16 
  at CommandLineArgsHandler.Start () [0x000c9] in C:\Games\Rauf Tur\Assets\CommandLineArgsHandler.cs:103 
Source: 
Date: System.Collections.ListDictionaryInternal
InnerException: 

25.11.2022 17:59:01: session closed successfully with response:StatusCode: 202, ReasonPhrase: 'Accepted', Version: 1.1, Content: System.Net.Http.StreamContent, Headers:
{
Server: nginx/1.23.1
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/7.4.25
Cache-Control: no-cache, private
Date: Fri, 25 Nov 2022 15:59:01 GMT
X-Debug-Token: c4ebc5
X-Debug-Token-Link: http://212.56.202.77/_profiler/c4ebc5
X-Robots-Tag: noindex
Content-Type: text/html; charset=UTF-8
}
25.11.2022 17:59:05: Message: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.

StackTrace: 

Server stack trace: 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.CreateConnection () [0x00036] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.HostConnectionPool.GetConnection () [0x000bb] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpConnectionPool.GetConnection (System.String host, System.Int32 port) [0x0005c] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.Tcp.TcpClientTransportSink.ProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg, System.Runtime.Remoting.Channels.ITransportHeaders requestHeaders, System.IO.Stream requestStream, System.Runtime.Remoting.Channels.ITransportHeaders& responseHeaders, System.IO.Stream& responseStream) [0x00025] in <f4b659e5df5b40fda5b047a02f25c201>:0 
  at System.Runtime.Remoting.Channels.BinaryClientFormatterSink.SyncProcessMessage (System.Runtime.Remoting.Messaging.IMessage msg) [0x0006c] in <f4b659e5df5b40fda5b047a02f25c201>:0 

Exception rethrown at [0]: 
  at (wrapper managed-to-native) System.Object.__icall_wrapper_mono_remoting_wrapper(intptr,intptr)
  at (wrapper remoting-invoke) IMarketingGameResult.IMarketingGameResult.Prize(string)
  at Client.SendToServer (System.String[] data) [0x0003c] in C:\Games\Rauf Tur\Assets\Scripts\Classes\Client.cs:17 
  at QuitGame.Update () [0x000b5] in C:\Games\Rauf Tur\Assets\Scripts\QuitGame.cs:48 
Source: 
Date: System.Collections.ListDictionaryInternal
InnerException: 

